README.md

```diff
@@ -115,17 +116,17 @@ fmt.Println(formattedStr)
 
 ### Interpreting eris stack traces
 
-Errors created with this package contain stack traces that are managed automatically. They're currently mandatory when creating and wrapping errors but optional when printing or logging. The stack trace and all wrapped layers follow the same order as Go's `runtime` package, which means that the root cause of the error is shown first.
+Errors created with this package contain stack traces that are managed automatically. They're currently mandatory when creating and wrapping errors but optional when printing or logging. By default, the stack trace and all wrapped layers follow the opposite order of Go's `runtime` package, which means that the original calling method is shown first and the root cause of the error is shown last.
```

Comment:
Should we explain the philosophy behind it here?

Reply:
i'm not sure i know what to write for that. any ideas?

Reply:
Two points: familiarity with pkg errors and following standard definition of the trace. Feel free to skip it, it isn't adding anything